sdk/keyvault/azure-keyvault-administration/tests/recordings/test_backup_client.test_full_backup_and_restore.yaml

```diff
@@ -113,7 +113,7 @@ interactions:
       cache-control:
       - no-cache
       content-length:
-      - '294'
+      - '138'
```

Comment:
That's weird: same content, different content-length :suspect:

Reply:
This must be a relic from rebasing -- I'll re-record and look out for similar funny business